articles/service-fabric/service-fabric-containers-overview.md

```diff
@@ -46,8 +46,8 @@ Compared to virtual machines, containers have the following advantages:
 Service Fabric supports the deployment of Docker containers on Linux, and Windows Server containers on Windows Server 2016 and later, along with support for Hyper-V isolation mode.
 
 Container runtimes compatible with ServiceFabric:
-- Linux: Mirantis Container Runtime + Ubuntu
-- Windows: Mirantis Container Runtime + Windows Server 2019/2022
+- Windows: Mirantis Container Runtime with WS2022 and DockerEE with WS2019

```

Comment:
Mabe you can add sub-bullets to Windows as follows for better readability?
Windows Server 2022: Mirantis Container Runtime
Windows Server 2019/2016: DockerEE